### PR TITLE
fix prometheusMetrics export contains " bug

### PR DESCRIPTION
**Pull request by hangc0276**
when i use Oracle JDK 10.0.2，the broker metrics `jvm_info` will export as follow: 
```
# TYPE jvm_info gauge
jvm_info{cluster="pulsar-cluster-zk",version="10.0.2+13",vendor=""Oracle Corporation"",runtime="Java(TM) SE Runtime Environment"} 1.0
```
and then prometheus will failed when parsing `""Oracle Corporation""`
image

The Reason is when using `io.prometheus:simpleclient:0.5.0` to collect jvm info, it returns jvm vendor with `"Oracle Corporation"`, and in the Pulsar source code, it adds `"` in both sides, so it comming with `vendor=""Oracle Corporation""`, and prometheus parse failed.

### CODE REVIEW

**Comment by jiazhai** [comment]
run cpp tests

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java [hunk 1]
```diff
@@ -95,10 +95,17 @@ private static void generateSystemMetrics(SimpleTextOutputStream stream, String
                 stream.write(sample.name);
                 stream.write("{cluster=\"").write(cluster).write('"');
                 for (int j = 0; j < sample.labelNames.size(); j++) {
+                    String labelValue = sample.labelValues.get(j);
+                    if(labelValue != null &&
+                            (labelValue.startsWith("\"") ||
```
Comment (sijie) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java]: I don't think we need to check `startsWith` and `endsWith`.

We can just simplify the logic as follows:

```
if (labelValue != null) {
    labelValue = labelValue.replace("\"", "\\\"");
}
```

Comment (hangc0276) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java]: ok, that's more simple! I improve the code.

**Comment by hangc0276** [comment]
run java8 tests
run integration tests

**Comment by hangc0276** [comment]
run java8 tests

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java [hunk 2]
```diff
@@ -95,10 +95,15 @@ private static void generateSystemMetrics(SimpleTextOutputStream stream, String
                 stream.write(sample.name);
                 stream.write("{cluster=\"").write(cluster).write('"');
                 for (int j = 0; j < sample.labelNames.size(); j++) {
+                    String labelValue = sample.labelValues.get(j);
+                    if(labelValue != null){
```
Comment (sijie) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java]: nit:
```suggestion
                    if (labelValue != null) {
```

**Comment by sijie** [comment]
run java8 tests

**Comment by hangc0276** [comment]
run java8 tests

**Comment by hangc0276** [comment]
run java8 test

**Comment by jiazhai** [comment]
run java8 test

**Comment by sijie** [comment]
run java8 tests

**Comment by hangc0276** [comment]
run java8 tests

**Comment by hangc0276** [comment]
run java8 tests

**Pull request merged**